main.js

```diff
@@ -195,6 +196,73 @@ function createWindow() {
                         });
                     },
                 },
+                {type: 'separator'},
+                {
+                    label: 'Export database',
+                    click() {
+                        var options = {
+                            title: 'Export DB to file',
+                            defaultPath : 'time_to_leave',
+                            buttonLabel : 'Export',
+                  
+                            filters :[
+                              {name: '.ttldb', extensions: ['ttldb',]},
+                              {name: 'All Files', extensions: ['*']}
+                            ]
+                        };
+                        let response = dialog.showSaveDialogSync(options);
+                        if (response) {
+                            exportDatabaseToFile(response);
+                            dialog.showMessageBox(BrowserWindow.getFocusedWindow(),
+                                {
+                                    title: 'Time to Leave',
+                                    message: 'Database export',
+                                    type: 'info',
+                                    icon: iconpath,
+                                    detail: '\Okay, database was exported.'
+                                });
+                        }
+                    },
+                },
+                {
+                    label: 'Import database',
+                    click() {
+                        var options = {
+                            title: 'Import DB from file',
+                            buttonLabel : 'Import',
+                  
+                            filters :[
+                              {name: '.ttldb', extensions: ['ttldb',]},
+                              {name: 'All Files', extensions: ['*']}
+                            ]
+                        };
+                        let response = dialog.showOpenDialogSync(options);
+                        if (response) {
+                            const options = {
+                                type: 'question',
+                                buttons: ['Cancel', 'Yes, please', 'No, thanks'],
+                                defaultId: 2,
+                                title: 'Import database',
+                                message: 'Are you sure you want to import a database? It may override your information.',
```

Comment:
```suggestion
                                message: 'Are you sure you want to import a database? It will override any current information.',
```
It will override, regardless if there is or there isn't, wouldn't it?